test/DotNetEnv.Tests/EnvTests.cs

```diff
@@ -62,5 +62,14 @@ public void LoadPathArgsTest()
             DotNetEnv.Env.Load("./.env2", false, false, false);
             Assert.Equal(Environment.GetEnvironmentVariable("UNICODE"), "'\\u00ae \\U0001F680 日本'");
         }
+
+        [Fact]
+        public void LoadNoClobberTest()
+        {
+            var expected = "totally the original value";
+            Environment.SetEnvironmentVariable("URL", expected);
+            DotNetEnv.Env.Load(false, false, false, false);
+            Assert.Equal(Environment.GetEnvironmentVariable("URL"), expected);
```

Comment:
There are other tests that do implicitly test the old behavior of clobbering, but do you think that perhaps one additional setup + assertion inside this test to explicitly test the "true" behavior could be added?